modules/private-clusters-setting-api-private.adoc

```diff
@@ -50,7 +50,7 @@ lk4pj-worker-us-east-1b-zgpzg   running   m4.xlarge   us-east-1   us-east-1b   1
 ----
 +
 You modify the control plane machines, which contain `master` in the name, in the following step.
-
+//todo: assess for CPMS impact.
```

Comment:
Post-installation configuration > Configuring a private cluster > [Restricting the API server to private](https://53397--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-private-cluster.html#private-clusters-setting-api-private_configuring-private-cluster)

Reply:
This procedure is for AWS and Azure, so I am thinking:
- add a statement like "for clusters that use the CPMS, refer to CPMS docs" 
- adding a version of this topic to the CMPS docs
- add an "Additional Resources" link to CMPS docs

Reply:
Yep, sounds good